src/sql/azdata.proposed.d.ts

```diff
@@ -3206,11 +3206,18 @@ declare module 'azdata' {
 		customAction = 1
 	}
 
+	export enum ColumnSizingMode {
+		ForceFit = 0,	// all columns will be sized to fit in viewable space, no horiz scroll bar
+		AutoFit = 1,	// columns will be ForceFit unless there are too many.  If too many component will revert to default column sizing
```

Comment:
>too many. [](start = 60, length = 9)

What does "too many" mean? Is there a minimum size specified somewhere?

Reply:
Ok I see the commend in the implementation in table.component.ts - that's fine but it might be nice to somehow convey that here as it currently isn't clear how the limit is decided

---
In reply to: [304167009](https://github.com/microsoft/azuredatastudio/pull/6387#discussion_r304167009) [](ancestors = 304167009)

Reply:
I'll add the specific behavior as a comment.